src/components/data/data-utils.js

```diff
@@ -467,7 +467,7 @@ export const EntityUploader = ({ onSuccess, onDismiss, namespace, name, entityTy
             }
           })
         ]),
-        ((currentFile && entityTypeAlreadyExists) || _.includes(recordType, entityTypes)) && div({
+        ((isGoogleWorkspace && currentFile && entityTypeAlreadyExists) || (!isGoogleWorkspace && _.includes(recordType, entityTypes))) && div({
```

Comment:
Just double checking -- did you want this to be `!isGoogleWorkspace` rather than `isGoogleWorkspace`?

Reply:
One IS `!isGoogleWorkspace` and the other `isGoogleWorkspace`, isn't it?

Reply:
gotcha! Just wanted to make sure that that logic was intended!